python/src/aiconfig/editor/client/src/components/prompt/prompt_metadata/PromptMetadataSchemaRenderer.tsx

```diff
@@ -0,0 +1,46 @@
+import SettingsPropertyRenderer, {
+  SetStateFn,
+} from "../../SettingsPropertyRenderer";
+import { GenericPropertiesSchema } from "../../../utils/promptUtils";
+import { JSONObject, JSONValue } from "aiconfig";
+import { memo, useMemo } from "react";
+import { debounce } from "lodash";
+
+type Props = {
+  schema: GenericPropertiesSchema;
+  metadata?: JSONObject;
+  onUpdatePromptMetadata: (metadata: Record<string, unknown>) => void;
+};
+
+export default memo(function ModelSettingsSchemaRenderer({
+  schema,
+  metadata,
+  onUpdatePromptMetadata,
+}: Props) {
+  const debouncedConfigUpdate = useMemo(
+    () =>
+      debounce(
+        (newMetadata: JSONObject) => onUpdatePromptMetadata(newMetadata),
+        250
```

Comment:
Don't we have a constant val for this in utils?

Reply:
Whoops, yep